src/execution/operator/scan/physical_table_scan.cpp

```diff
@@ -136,6 +136,7 @@ string PhysicalTableScan::GetName() const {
 
 InsertionOrderPreservingMap<string> PhysicalTableScan::ParamsToString() const {
 	InsertionOrderPreservingMap<string> result;
+	result["Function"] = StringUtil::Upper(function.name);
```

Comment:
This keeps the operator function name and moves it into the extra info?

Reply:
The changes to the ProfilingNode caused us to lose the `RANGE` name on the nodes, turning it into `PHYSICAL TABLE SCAN` instead. This adds back the name so it can be identified